spec/zeus/api_client_spec.rb

```diff
@@ -19,7 +19,8 @@
 
   let (:zeus_client) do
     Zeus::APIClient.new({
-      access_token: 'fake_access_token'
+      access_token: 'fake_access_token',
+      endpoint: 'luke.skywalker.com'
```

Comment:
@victorfonsec4 Thank you for the PR.

For the test, would u mind to add test code something like following?:

```ruby
# I haven't checked this code, so just an example.
describe '#initialize' do
    context 'endpoint starts with http' do
      it 'replace http to https' do
        fake_zeus_client = Zeus::APIClient.new({
          access_token: 'fake_access_token',
          endpoint: 'http://fake-ciscozeus.io'
        })
        expect(fake_zeus_client.instance_variable_get(:@endpoint)).to eq('https://fake-ciscozeus.io')
      end
    end
    context 'endpoint without schema' do
      it 'add https to the start of the url' do
        fake_zeus_client = Zeus::APIClient.new({
          access_token: 'fake_access_token',
          endpoint: 'fake-ciscozeus.io'
        })
        expect(fake_zeus_client.instance_variable_get(:@endpoint)).to eq('https://fake-ciscozeus.io')
      end
    end
  end
```